zcash_primitives/src/transaction/builder.rs

```diff
@@ -201,36 +201,40 @@ impl Progress {
 #[derive(Clone, Copy)]
 pub enum BuildConfig {
     Standard {
-        sapling_anchor: sapling::Anchor,
-        orchard_anchor: orchard::Anchor,
+        sapling_anchor: Option<sapling::Anchor>,
+        orchard_anchor: Option<orchard::Anchor>,
```

Comment:
(Asking so I can keep track of the design:) Why are the anchors now optional again? (And why the change in this PR - what about the `orchard` flag necessitates this?)

Reply:
So, the anchors are optional so that we don't have to depend on the `orchard` crate in `zcash_client_backend`, since the feature flagging doesn't yet extend to `zcash_primitives`. We will likely want to do that work in `zcash_primitives`, but that's a bigger job that I didn't want to tackle yet. So making these optional is the compromise.